packages/default-reporter/src/index.ts

```diff
@@ -71,23 +74,32 @@ export default function (
   }
 }
 
-export function toOutput$ (
-  opts: {
-    streamParser: object
-    reportingOptions?: {
-      appendOnly?: boolean
-      logLevel?: LogLevel
-      outputMaxWidth?: number
-      streamLifecycleOutput?: boolean
-      throttleProgress?: number
-    }
-    context: {
-      argv: string[]
-      config?: Config
-    }
+export function toOutput$ (opts: {
+  streamParser: object
+  reportingOptions?: {
+    appendOnly?: boolean
+    logLevel?: LogLevel
+    outputMaxWidth?: number
+    streamLifecycleOutput?: boolean
+    throttleProgress?: number
+  }
+  context: {
+    argv: string[]
+    config?: Config
   }
-): Rx.Observable<string> {
+}): Rx.Observable<string> {
   opts = opts || {}
+  let filterLog: undefined | ((log: logs.Log) => boolean)
+  if (opts.context.config != null) {
+    const pnpmOpts = opts.context.config
+    const { ignorePnpmfile, lockfileDir, dir } = pnpmOpts
+    if (!ignorePnpmfile) {
+      const hooks: any = requireHooks(lockfileDir ?? dir, pnpmOpts)
```

Comment:
I don't think this library should directly access the pnpmfile. The filter function should probably be passed in to the reporter